ci-operator/step-registry/openstack/provision/securitygroups/openstack-provision-securitygroups-ref.yaml

```diff
@@ -14,5 +14,25 @@ ref:
     - name: ADDITIONAL_SECURITY_GROUP_RULES
       default: ""
       documentation: Name of the security group to apply
+    - name: CONFIG_TYPE
+      documentation: |
+        The type of config for the environment to deploy.
+
+        * 'minimal' - Configure the install-config with the minimal options to make it work on
+                      our tests, which requires the usage of floating IPs. This config type doesn't
+                      require a proxy server to be deployed.
+        * 'proxy' - Configure the install-config to use a pre-created restricted network (BYON) and
+                   a Cluster Proxy.
+    - name: MACHINES_SUBNET_v4_RANGE
+      default: "192.168.25.0/24"
+      documentation: |-
+        CIDR of the IPv4 Machines Subnet
+    - name: MACHINES_SUBNET_v6_RANGE
```

Comment:
```suggestion
    - name: MACHINES_SUBNET_V6_RANGE
```

Reply:
I have the same observation, but to use the same v6 and v4 machine subnet range I need to use the same env variable defined [here](https://github.com/openshift/release/blob/master/ci-operator/step-registry/openstack/conf/installconfig/openstack-conf-installconfig-ref.yaml) . However, I'll change the strategy and get only the v4 machine subnet for minimal config type that I need in this PR.